doc/source/rllib/index.rst

```diff
@@ -66,103 +66,115 @@ for a few training iterations, then perform a single evaluation loop
+        | 3) `Independent learning <https://github.com/ray-project/ray/blob/master/rllib/examples/multi_agent_independent_learning.py>`_
+          of neutral/co-existing agents.
 
-.. container::
 
-    **External simulators** connecting to RLlib from the outside (e.g.
-    via http(s)): Don't have your simulation running as a gym.Env in python?
-    No problem, RLlib supports an external environment API and comes with a pluggable,
-    off-the-shelve client/server setup that allows you to run 100s of independent
-    simulators on the "outside" (e.g. a Windows cloud) connecting to a central RLlib
-    Policy-Server that learns and serves actions. Alternatively, actions can be computed
-    on the client side to save on network traffic.
+.. container:: clear-both
 
-.. .. container:: clear-both
+    .. container:: buttons-float-left
 
-..    .. container:: buttons-float-left
+        .. image:: ../images/rllib/sigils/rllib-sigil-external-simulators.svg
+            :width: 100
+            :target: https://github.com/ray-project/ray/tree/master/rllib/examples/serving
 
-..        .. image:: ../images/rllib/sigils/offline-rl.svg
-..            :width: 120
+    .. container::
 
-.. container::
+        **External simulators**: Don't have your simulation running as a gym.Env in python?
+        No problem! RLlib supports an external environment API and comes with a pluggable,
+        off-the-shelve client/server setup that allows you to run 100s of independent
+        simulators on the "outside" (e.g. a Windows cloud) connecting to a central RLlib
+        Policy-Server that learns and serves actions. Alternatively, actions can be computed
+        on the client side to save on network traffic.
 
-    **Offline (batch) RL and imitation learning (behavior cloning)** using historic data:
-    If you don't have a simulator for your particular problem, but tons of historic data
-    recorded by a legacy (maybe non-RL/ML system), this branch of reinforcement learning
-    is for you! RLlib's comes with several offline RL algorithms (*CQL*, *MARWIL*, and *DQfD*),
-    allowing you to either purely behavior-clone the historic system (the one that recorded
-    your historic data) or learn how to improve over that system.
 
+.. container:: clear-both
 
-In-Depth Documentation
-----------------------
+    .. container:: buttons-float-left
+
+        .. image:: ../images/rllib/sigils/rllib-sigil-offline-rl.svg
+            :width: 100
+            :target: https://github.com/ray-project/ray/blob/master/rllib/examples/offline_rl.py
+
+    .. container::
+
+        **Offline RL and imitation learning/behavior cloning**: You don't have a simulator
+        for your particular problem, but tons of historic data recorded by a legacy (maybe
+        non-RL/ML) system? This branch of reinforcement learning is for you!
+        RLlib's comes with several offline RL algorithms (*CQL*, *MARWIL*, and *DQfD*),
```

Comment:
shit (note to myself), I really need to get to these algorithms :) 💪

Reply:
:D